cli/pkg/kctrl/cmd/app/init.go

```diff
@@ -149,7 +151,9 @@ func (o *InitOptions) writeAppFile(appBuild *buildconfigs.AppBuild) error {
 	o.ui.PrintInformationalText("Successfully updated app-build.yml\n")
 	o.ui.PrintInformationalText("Successfully updated app.yml\n")
 	o.ui.PrintHeaderText("\n**Next steps**")
-	o.ui.PrintInformationalText("Created files can be consumed in following ways:\n1. Optionally, use 'kctrl dev deploy' to iterate on the app and deploy locally.\n2. Use 'kctrl app release' to release the app.\n")
+	o.ui.PrintInformationalText("Created files can be consumed in following ways:\n" +
+		"1. Optionally, use `dev deploy` command to iterate on the app and deploy locally.\n" +
+		"2. Use `app release` command to release the app.\n")
```

Comment:
Let's remove this since the command is not mounted?

Reply:
Done